3rdparty/cldr/README.md

```diff
@@ -0,0 +1,47 @@
+# Unicode CLDR Project
```

Comment:
This file isn't in the CLDR 38 release (http://unicode.org/Public/cldr/38/), so I think we should remove it.

My usual procedure is: `git rm -fr 3rdparty && mkdir -p 3rdparty/cldr && (cd 3rdparty/cldr && unzip ~/Downloads/core.zip) && git add 3rdparty && git commit -m "Update CLDR to release blabla"`

(the key point being that all the files from the previous release only get carried over if they're also in the new release)